### DISCUSSION TITLE
Trying to use Deluge experimental support

### DISCUSSION THREAD

**Discussion by penguincrest** [jesec/flood#545] [General]
I'm trying to add Deluge, and I have Deluge set up without a password.  Every time I try to add it I just get "unknown error has occured."  I tried to find documentation on here to find proper Deluge instructions but I can't seem to find anything.  Other than the IP and port number, what else needs to be entered as far as username/password?

**Comment by jarreboum** [comment]
late answer, but you need to connect to the daemon, not the web interface. By default port 58846, and the username and passwords are in deluged/config/auth. username:password:10
